README.rst

```diff
@@ -1,35 +1,41 @@
 mando: CLI interfaces for Humans!
 =================================
 
-.. image:: https://img.shields.io/travis/rubik/mando/master.svg
+.. image:: https://img.shields.io/travis/rubik/mando
     :alt: Travis-CI badge
     :target: https://travis-ci.org/rubik/mando
 
-.. image:: https://img.shields.io/coveralls/rubik/mando/master.svg
+.. image:: https://img.shields.io/coveralls/rubik/mando
     :alt: Coveralls badge
     :target: https://coveralls.io/r/rubik/mando
 
-.. image:: https://img.shields.io/pypi/v/mando.svg
+.. image:: https://img.shields.io/pypi/implementation/mando?label=%20&logo=python&logoColor=white
+    :alt: PyPI - Implementation
+
+.. image:: https://img.shields.io/pypi/v/mando
     :alt: Latest release
     :target: https://pypi.python.org/pypi/mando
 
-.. image:: https://img.shields.io/pypi/format/mando.svg
+.. image:: https://img.shields.io/pypi/l/mando
+    :alt: PyPI - License
+    :target: https://pypi.org/project/mando/
+
+.. image:: https://img.shields.io/pypi/pyversions/mando
+    :alt: PyPI - Python Version
+    :target: https://pypi.org/project/mando/
+
+.. image:: https://img.shields.io/pypi/format/mando
     :alt: Download format
     :target: http://pythonwheels.com/
 
-.. image:: https://img.shields.io/pypi/l/mando.svg
-    :alt: Mando license
-    :target: https://pypi.python.org/pypi/mando/
 
 mando is a wrapper around ``argparse``, and allows you to write complete CLI
 applications in seconds while maintaining all the flexibility.
 
 Installation
 ------------
 
-Mando is tested across all Python versions from **Python 2.6** to **Python
```

Comment:
In case it's unclear, I added badges that relay this information (pyversions and implementation).